### ISSUE TITLE
HOST SERVER

### ISSUE THREAD

**Issue by devonpmack** [devonpmack/simulate_natural_selection#33]
1. Make it a server, compile for production with webpack and then host with node? #17 
2. Host it #14 
3. #15 
4. Domain name #16

**Comment by DavidHZhu** [comment]
Is there anything else we need for the mvp? Cant heroku connect to a github repo and rebuild when we make changes? Basically is there any point to not doing this now rather than later if we have a working mvp 🤔 @devonpmack @Billzzhang @justinpaoli

**Comment by devonpmack** [comment]
Yeah we have MVP we can do this anytime, just have to setup webpack and node for production
